src/AppBundle/Entity/Report/Report.php

```diff
@@ -834,6 +843,24 @@ public function setAgreedBehalfDeputyExplanation($agreedBehalfDeputyExplanation)
         return $this;
     }
 
+
+    /**
+     * @return Document[]
+     */
+    public function getDocuments()
+    {
+        return $this->documents;
+    }
+
+    /**
+     * @param Document[] $documents
```

Comment:
in the client, no collections, but arrays. not sure we want to use "Document[]" or "array" as annotation ?